### PR TITLE
Proper fix for {"reload": "now"}

### PR DESCRIPTION
**Pull request by Aasim-A**

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -2,23 +2,41 @@ const FIRSTPAGE = require('./firstpage.js');
 const NONFIRSTPAGE = require('./nonfirstpage.js');
 const util = require('./util.js');
 
-const DEFAULT_USER_AGENT = 'Mozilla/5.0 (compatible; bingbot/2.0; +http://www.bing.com/bingbot.htm)'; // eslint-disable-line max-len
+const DEFAULT_USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.117 Safari/537.36'; // eslint-disable-line max-len
+const DEFAULT_X_YOUTUBE_CLIENT_NAME = '1';
+const DEFAULT_X_YOUTUBE_CLIENT_VERSION = '1.20200613.00.00';
```
Comment (TimeForANinja) [inline lib/main.js:7]: kinda dislike hardcoding all this
still thinking about a way to get this information from server responses

Comment (Aasim-A) [reply lib/main.js:7]: If that could be done, that would be ideal, but if not, I think this is the more likely solution

**Comment by trustedtomato** [comment]
I agree with @Aasim-A, I think the most long-termly supported method is to mimic a real user.
But repeating the code `optionsOrCallback = { limit: 100, headers: {
      'user-agent': DEFAULT_USER_AGENT,
      'x-youtube-client-name': DEFAULT_X_YOUTUBE_CLIENT_NAME,
      'x-youtube-client-version': DEFAULT_X_YOUTUBE_CLIENT_VERSION,
    } };` multiple times is not very beautiful, and [line 36](https://github.com/TimeForANinja/node-ytpl/pull/55/files#diff-16b7d75b70953cbf4755508170f2f24cR36) and [line 38](https://github.com/TimeForANinja/node-ytpl/pull/55/files#diff-16b7d75b70953cbf4755508170f2f24cR38) should be `if` instead of `else if`, although a simple `Object.assign` would do it (see #61), I find that much more readable.

**Comment by Aasim-A** [comment]
@trustedtomato Cool! that's certainly a better implementation, closing this PR to get yours merged

**Pull request closed without merging**